cssdb.json

```diff
@@ -543,6 +556,28 @@
     ],
     "vendors_implementations": 0
   },
+  {
+    "id": "float-clear-logical-values",
+    "title": "Logical Values in float and clear",
+    "description": "Flow-relative (inline-start and inline-end) values for float and clear",
+    "specification": "https://www.w3.org/TR/css-logical-1/",
+    "stage": 2,
+    "browser_support": {
+      "and_ff": "55",
+      "firefox": "55"
+    },
+    "docs": {
+      "mdn": "https://developer.mozilla.org/en-US/docs/Web/CSS/float#browser_compatibility"
+    },
+    "example": "span:first-child {\n  float: inline-start;\n}",
+    "polyfills": [
+      {
+        "type": "PostCSS Plugin",
+        "link": "https://github.com/csstools/postcss-plugins/tree/main/plugins/postcss-logical"
+      }
+    ],
+    "vendors_implementations": 1
+  },
```

Comment:
Will need extra code in postcss-preset-env but this sub feature never landed in other browsers.

This was causing the entire logical properties and values feature to seem unsupported in browsers.

Better to split it and deal with the fallout downstream (imo).

Reply:
Completely agreed